client/scanners.go

```diff
@@ -117,17 +141,45 @@ const (
 	ScannerLabelCreatableOnTool = "creatable-on-tool"
 )
 
+type ListActiveScannersInput struct {
+	Types  []ScannerType
+	Labels string
+	Name   string
+	Limit  int
+}
+
+func (i *ListActiveScannersInput) prepareRequestQueryParameters() ScannersSearchParams {
+	var scannerTypes = make([]string, 0)
+
+	if i.Types == nil || len(i.Types) == 0 {
+		for _, t := range ScannerTypes() {
```

Comment:
`len(i.Types) == 0` directly to check for empty or nil slices, there is no need to add another nil check.